pyproject.toml

```diff
@@ -19,6 +19,9 @@ black = {version = "^22.3.0", allow-prereleases = true}
 pylint = "^2.15.3"
 pycodestyle = "^2.9.1"
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
Oh seems like this is the new way to structure dev dependecies. We should move all the dependencies from the [tool.poetry.dev-dependencies] over here. (For all our repositories) :100:

Reply:
Agreed, I'll move all dev dependencies to the new section. Yes, we need to do the same for other repos 👍